fem/pfespace.hpp

```diff
@@ -101,6 +123,19 @@ class ParFiniteElementSpace : public FiniteElementSpace
    // Auxiliary method used in constructors
    void ParInit(ParMesh *pm);
 
+   void CommunicateGhostOrder(Array<VarOrderElemInfo> & pref_data);
```

Comment:
Document all the new methods in this file?

Reply:
Done in fc63c5e90b5ace705c64319123d3715d1dbe250f